Resources/views/Layouts/macros.html.twig

```diff
@@ -160,9 +160,9 @@
         </tr>
     {% endif %}
     {% for i in 0..loopMax - 1 %}
-        <tr class="{% if i % 2  == 0 %}even{% else %}odd{% endif %}">
+        <tr {% if colorAltern == 'line' %}class="{% if i % 2  == 0 %}even{% else %}odd{% endif %}"{% endif %}>
         {% for index,hour in hours_range %}
-            <td class="timegrid-column">{% spaceless %}
+            <td class="timegrid-column{% if colorAltern == 'column' %}{% if index % 2  == 0 %} even{% else %} odd{% endif %}{% endif %}">{% spaceless %}
```

Comment:
odd and even check is repeated above. you can do it once and put the result in a variable